pkg/storagepool/storagepool_template_engine.go

```diff
@@ -19,11 +19,11 @@ package storagepool
 
 import (
 	"fmt"
-	"strings"
 	"github.com/ghodss/yaml"
 	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/engine"
 	"github.com/openebs/maya/pkg/util"
+	"strings"
 )
```

Comment:
Same as above comment

Reply:
@prateekpandey14 I just ran ```go fmt``` on the said file and got the above result. I don't think there would be any other change in that particular file in terms of formatting, I will revert these changes, but then is the original issue valid?

Reply:
So I ran ```goimport``` and revert the changes, but now Better Code Hub is panicking, any idea why?

Reply:
Ok, got it, doesn't look like the files need any changes, closing this PR. Apologies for the noise. Probably the original issue should be closed as well.